controllers/controllers.js

```diff
@@ -0,0 +1,11 @@
+const { fetchAllTopics } = require("../models/models")
+
+const getAllTopics = (req, res, next) => {
+    console.log("in controller")
```

Comment:
Remove any console.logs that are not relevant anymore before merging next time. Consider the code you push as going straight into the final product. This log tells me that you still have bugs you're trying to trace and fix, which would make me think that the code isn't complete yet if I was reviewing this as your senior. If you've pushed this code, you should be confident that it works and you won't need these logs anymore.